.github/workflows/CI.yml

```diff
@@ -45,13 +45,17 @@ jobs:
           # be able to properly deal with PRs / merges
           fetch-depth: 2
       - name: "Set up Julia"
+        id: setup-julia
         uses: julia-actions/setup-julia@v2
         with:
           version: ${{ matrix.julia-version }}
       - name: "Cache artifacts"
         uses: julia-actions/cache@v2
       - name: "Build package"
         uses: julia-actions/julia-buildpkg@v1
+      - name: "workaround libstdc++ issue for julia 1.6"
+        if: matrix.julia-version == '1.6' # && runner.os == 'Linux'
+        run: rm -f ${{ steps.setup-julia.outputs.julia-bindir }}/../lib/julia/libstdc++.so.6
```

Comment:
In the runner this somehow ends up as `rm -f /../lib/julia/libstdc++.so.6` so the `julia-bindir` is empty. But at least on the surface I see no mistake??

Any hints @benlorenz @lgoettgens ?